### ISSUE TITLE
Datasets download

### ISSUE THREAD

**Issue by mathamateur** [Nicolas-BZRD/llm-distillation#2]
Hello! I would like to reproduce the results of your work, but I am not able to download generated datasets with git lfs. When I had tried `git lfs fetch` after cloning the repo, I got the following error:
Fetching main
Git LFS: (0 of 19 files) 0 B / 490.31 MB
batch response: This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.
error: failed to fetch some objects from 'https://github.com/Nicolas-BZRD/llm-distillation.git/info/lfs'

I searched for generated datasets here https://huggingface.co/Nicolas-BZRD as well, but I didn't find the proper dataset.

Could you please upload generated datasets on huggingface or help me to deal with lfs?

**Comment by Nicolas-BZRD** [comment]
Hi Mathamateur,

Thank you for your interest in the project.

I have uploaded the dataset on my [HuggingFace profile](https://huggingface.co/Nicolas-BZRD) (Git LFS is free until 5GB), and the datasets for this project are prefixed by 'uld_loss_...'.

Note: To load the dataset during training, you should pass the path of the loader file associated with the dataset, and not directly the dataset itself. For example, to try with SQuAD, you should pass the file [squad.py](https://github.com/Nicolas-BZRD/llm-distillation/blob/main/datasets/loader/squad.py) to the parameter --dataset.file of the [llm-recipes fine-tuning script](https://github.com/Nicolas-BZRD/llm-recipes). This file ensures that the exact same number of few-shot examples and templates are passed to the teacher to generate the exact same sentences and associated logits.
